tests/modeldiffs/librispeech_deepspeech/compare.py

```diff
@@ -54,30 +56,24 @@ def sd_transform(sd):
       else:
         out[k] = sd[k]
     elif 'LSTM' in ''.join(k):
-      if '_hh_' in ''.join(k):
-        chunks = sd[k].chunk(4)
-        for t, c in zip(['hi', 'hf', 'hg', 'ho'], chunks):
-          out[k[:-1] + ('LSTM',
-                        f'LSTMSequenceEncoder_{1 if "reverse" in k[-1] else 0}',
-                        'cell',
-                        t,
-                        k[-1].split('_')[0])] = c
-      elif '_ih_' in ''.join(k):
-        chunks = sd[k].chunk(4)
-        for t, c in zip(['ii', 'if', 'ig', 'io'], chunks):
-          out[k[:-1] + ('LSTM',
-                        f'LSTMSequenceEncoder_{1 if "reverse" in k[-1] else 0}',
-                        'cell',
-                        t,
-                        k[-1].split('_')[0])] = c
+      l = out.get(k[:-1], dict())
+      l[k[-1]] = sd[k]
+      out[k[:-1]] = l
     else:
       out[k] = sd[k]
   keys_to_del = []
+  updates = dict()
   for k in out:
-    if (k[-2] in ['ii', 'if', 'ig', 'io']) and k[-1] == 'bias':
-      other_bias = k[:-2] + ('h' + k[-2][1], 'bias')
-      out[other_bias] = out[other_bias] + out[k]
+    if isinstance(out[k], dict):
```

Comment:
nit: nbd if not, but is it possible to have a more descriptive check here? like checking for `k` being whatever the values of `k[:-1]` in the `elif 'LSTM' in ''.join(k):` above are? if not, can you add a comment?